seatunnel-connectors-v2/connector-http/src/main/java/org/apache/seatunnel/connectors/seatunnel/http/source/HttpSource.java

```diff
@@ -39,23 +32,22 @@
 import org.apache.seatunnel.connectors.seatunnel.common.source.AbstractSingleSplitReader;
 import org.apache.seatunnel.connectors.seatunnel.common.source.AbstractSingleSplitSource;
 import org.apache.seatunnel.connectors.seatunnel.common.source.SingleSplitReaderContext;
+import org.apache.seatunnel.connectors.seatunnel.http.config.HttpConfig;
+import org.apache.seatunnel.connectors.seatunnel.http.config.HttpParameter;
 
 import org.apache.seatunnel.shade.com.typesafe.config.Config;
 
 import com.google.auto.service.AutoService;
 
-import java.util.Map;
-import java.util.stream.Collectors;
-
 @AutoService(SeaTunnelSource.class)
 public class HttpSource extends AbstractSingleSplitSource<SeaTunnelRow> {
-    private final HttpSourceParameter parameter = new HttpSourceParameter();
+    private final HttpParameter httpParameter = new HttpParameter();
     private SeaTunnelRowType rowType;
     private SeaTunnelContext seaTunnelContext;
 
     @Override
     public String getPluginName() {
-        return "Http";
+        return "HttpSource";
```

Comment:
I think to keep `HTTP` it's better, We don't need too much redundant context

Reply:
But I think naming it this way would create some conflict with http sink, so if we keep it the same as before what's your advice about http sink name?